Main_Project/src/App.java

```diff
@@ -253,24 +268,37 @@ public static void readOne(String readProdID) {
 
     /** input '3' **/
     /** UPDATE METHOD **/
-    public static void update(String updateID, int updateQuant, float updateWholesale, float updatePrice, String updateSupplierID, Connection connection) throws IOException {
+    public static void update(String updateID, String updateQuant, String updateWholesale, String updatePrice, String updateSupplierID, Connection connection) throws IOException {
 
 		//UPDATE `yagni_inv_db`.`product` SET `quanity` = '1614', `Whole_sale` = '135.92', `Sale_cost` = '207.56', `vendor_id` = 'WBWVYLRD' WHERE (`product_id` = '001LORWG0PC0');
 
 		String sql_statement = "UPDATE `yagni_inv_db`.`product` SET ";
+		int count = 0;
 
-		String updateQuantString = Integer.toString(updateQuant);
-		String updateWholesaleString = Float.toString(updateWholesale);
-		String updatePriceString = Float.toString(updatePrice);
-
-		if (updateQuantString != ""){
-			sql_statement = sql_statement + "`quanity` = '" + updateQuantString + "'";
-		} else if (updateWholesaleString != "") {
-			sql_statement = sql_statement + ", `Whole_sale` = '" + updateWholesaleString + "'";
-		} else if (updatePriceString != "") {
-			sql_statement = sql_statement + ", `Sale_cost` = '" + updatePriceString + "'";
-		} else if (updateSupplierID != "") {
-			sql_statement = sql_statement + ", `vendor_id` = '" + updateSupplierID + "'";
+		if (updateQuant != null){
+			sql_statement = sql_statement + "`quanity` = '" + updateQuant + "'";
```

Comment:
just noticed i spelled quantity wrong

Reply:
it is spelled wrong because i copied code from mySQL workbench, where the column name is spelled wrong. resolving for now and will fix in the future as it doesn't break the code.

Reply:
Opened follow up ticket to adjust the table column name and queries in a future PR.  Jira ticket for tracking purposes - https://yagni-inc.atlassian.net/browse/YAG-33